src/ClientFactory.php

```diff
@@ -6,24 +6,28 @@
 
 class ClientFactory
 {
+    public const EVENT_TYPE_JOB_FAILED = 'job-failed';
+    public const VALID_EVENT_TYPES = [self::EVENT_TYPE_JOB_FAILED];
```

Comment:
misto toho by se krasne hodil enum ;)

Reply:
na to ti prdím když je tam jedna hodnota :)

Reply:
Hele, kdyby tady nebyl ten seznam validnich typu, asi by mi to vubec nedoslo. Takle to tam ale proste sviti.

No tak to nech, ted to funguje.. ale pocitej s tim, ze te budu chodit v noci strasit :)

Reply:
Fair enough :)